config/initializers/carrierwave.rb

```diff
@@ -25,7 +22,7 @@
         use_iam_profile: true,
         region: "us-east-2"
       }
-      config.asset_host = "https://#{ApplicationConfig['APP_DOMAIN']}/images"
```

Comment:
Will this affect existing forems in any way? Do we need to move any images that have already been uploaded?

Reply:
This will only affect Forems within the Forem-cloud. @jdoss and I are coordinating to roll this out properly. Dev.to and other self-hosted instance will work normally.

Reply:
Ok great, want to make sure we have a plan!!!!